mobile/src/main/java/org/openhab/habdroid/ui/WidgetAdapter.kt

```diff
@@ -860,7 +867,24 @@ class WidgetAdapter(
         }
 
         override fun onClick(view: View) {
-            connection.httpClient.sendItemCommand(boundWidget?.item, view.tag as String)
+            val button = view.tag as Widget
+            if (button.releaseCommand == null) {
+                // when there's a "releaseCommand", the normal "command" is sent in the onTouch event
+                button.command?.let { connection.httpClient.sendItemCommand(button.item, it) }
+            } else {
+                connection.httpClient.sendItemCommand(button.item, button.releaseCommand)
+            }
```

Comment:
Simplification suggestion:
```
// when there's a "releaseCommand", the normal "command" is sent in the onTouch handler
val command = button.releaseCommand
    ?: button.command
    ?: return
connection.httpClient.sendItemCommand(button.item, command)
```

Reply:
ohhhh some Kotliny goodness!

Reply:
I _think_ in this case, it's nicer to keep the if condition just for releaseCommand. It makes the code easier to understand at a glance:

if (button.releaseCommand == null) {
  // do this here
}

..... inside onTouch:
if (button.releaseCommand != null) {
  // this is basically the "else" part of the above
}

wdyt?

Reply:
I personally think the comment should be sufficient to explain that, but I don't have an overly strong opinion on this either.

Reply:
I removed the comment several commits ago :( Also I changed the null check to `isNullOrEmpty()`. See the latest version/comment below.